letsencrypt/cli.py

```diff
@@ -636,8 +638,12 @@ def preprocess_args(self, args):
                 self.verb = token
                 return reordered
 
-        self.verb = "run"
-        return args + ["run"]
+        if "--standalone" in args and "--installer" not in args and "-i" not in args:
```

Comment:
You could probably make the argument that we should remove my logic which keeps the verb as `run` if `-i INSTALLER` is supplied.  I'm fine either way there...